books/views.py

```diff
@@ -508,4 +509,17 @@ def birthdays(request: HttpRequest) -> HttpResponse:
     context = {
         'people_with_info': people_with_info,
     }
-    return render(request, 'books/stats/birthdays.html', context)
\ No newline at end of file
+    return render(request, 'books/stats/birthdays.html', context)
+
+
+def markdown_to_html(request: HttpRequest) -> HttpResponse:
+    '''Markdown to HTML'''
+    if request.method == 'POST':
+        markdown_text = request.body.decode('utf-8')
+        if markdown_text:
+            html_text = markdownify(markdown_text, custom_settings="book_description")
+            return HttpResponse(
+                content=html_text,
+                content_type='text/html',
+                headers={'Access-Control-Allow-Origin': '*'})
+    return HttpResponse(content="Bad request", content_type='text/plain', status=400)
```

Comment:
How about restructuring this code so that we deal with exceptions/invalid cases first and avoid if nesting. Also use error-specific messages like "GET not supported" or "missing body". It will be just extra few lines. 

Like here: https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement

Reply:
Fixed